test/example.md

```diff
@@ -1,20 +1,100 @@
 ## Interface: example
 
-### Method caption
-`example.methodName(a1, a2, a3)`
-First line description
-second line of multiline method description
-- `a1:number` - comment
-multiline comment
-for first argument
-- `a2` - array of string
-- `a3` - comment
+### One-liner function description
 
-Returns: string, comment
+Multi-line expanded function description. Note that all lines should have
+length of 80 or less characters
 
-Hint: message
+`methodName(num, str, arg, flag, arr, data, obj, cb)`
+  - `num: `[`<number>`] - argument description
+  - `str: `[`<string>`] - dash before and comma after types are optional
+  - `arg: `[`<Array>`]` | `[`<Date>`]` | `[`<Object>`] - argument can have multiple types
```

Comment:
I don't like the fact that this line is 89 characters long.

Reply:
Previously, all indentation was preserved but @lundibundi suggested removing it so that markdown will manage everything. IMO, all formatting in code comments should stay in docs.